docs/fundamentals/interface.rst

```diff
@@ -0,0 +1,105 @@
+Deepforge Interface
+===================
+The Deepforge editor interface is separated into six views for defining all of the necessary features of your desired project. The details of each interface tab are detailed below. You can switch to any of the views at any time by clicking the appropriate icon on the left side of the screen. In order, the tabs are:
+
++---------------+--------------------------+
+| |tabs|        | - Pipelines_             |
+|               | - Executions_            |
+|               | - Resources_             |
+|               | - Artifacts_             |
+|               | - `Custom Utils`_        |
+|               | - `Custom Serialization`_|
++---------------+--------------------------+
+
+.. |tabs| image:: interface_tabs.png
+
+Pipelines
+---------
+.. figure:: pipelines_tab.png
+    :align: center
+    :width: 75%
+
+In the initial view, all pipelines that currently exist in the project are displayed. New pipelines can be created using the red plus symbol in the bottom right. From this screen, existing pipelines can also be opened for editing, deleted, or renamed. Pipelines in this list are arranged automatically by the system and cannot be manually reordered in the current implementation.
```

Comment:
Wording changed.

The reordering was never a major issue during development for me, but I did find myself attempting to figure out the sorting method to force an ordering that I desired, but this was, of course, fruitless. Because of this, I expect some might waste a small amount of time on this, but I also expect that there will be relatively few users who will need to enforce a specific ordering. Outside of the tutorial context, I expect a sequential set of pipelines to be generally rare if not nonexistent.

You are probably right that this likely does not need mentioning, even offhand, in the documentation proper, but it may be a good addition to a FAQ-like page if one is added down the line (assuming the sorting is not changed before then).